src/components/SuccessPage/SuccessPage.js

```diff
@@ -35,7 +35,8 @@ const SuccessPage = () => {
         {formState.CustomerType === 'Scratchcard' ||
         formState.CustomerType === 'ClassPass' ? (
           <>
-            <p>We have sent you an email to acknowledge your application.</p>
+            <!-- <p>We have sent you an email to acknowledge your application.</p> -->
```

Comment:
Inside of React we need to use the JSX comment syntax which is...
```suggestion
            {/*  <p>We have sent you an email to acknowledge your application.</p>  */}
```